src/test/java/rx/internal/operators/SingleFromEmitterTest.java

```diff
@@ -173,6 +176,8 @@ public void call(SingleEmitter<Object> e) {
 
     @Test
     public void onSuccessThrows() {
+        final AtomicInteger onErrorCallsCounter = new AtomicInteger();
```

Comment:
Changing existing unit test is a sign of behavior change that may be incompatible with existing uses.